src/frontend/shared-components/src/components/sidecard/Sidecard.stories.js

```diff
@@ -15,6 +17,17 @@ const sideCard = {
   type: "blue"
 };
 
+const iconHeading = (
+  <div className="row">
+    <div className="side-card-row">
+      <div className="round-icon-wrapper">
+        <FaIdCard className="side-card-icon" />
+      </div>
+      <div className="side-card-title">About E-File Submission</div>
+    </div>
+  </div>
+);
```

Comment:
Is there no way to put some of this is SideCard and access it with a type or something? So that if you pass in <FaIdCard className="side-card-icon" /> to the img and a type of icon? Rather than needing to build this to use it?

Reply:
yeah we could conditionally render this if icon is present

Reply:
refactored!